pkg/globalmanager/downstream.go

```diff
@@ -145,11 +145,15 @@ func (dc *DownstreamController) syncIncrementalJob(eventType watch.EventType, jo
 
 	var trainNodeName string
 	var evalNodeName string
+	var deployNodeName string
 
 	ann := job.GetAnnotations()
 	if ann != nil {
 		trainNodeName = ann[AnnotationsKeyPrefix+string(sednav1.ILJobTrain)]
 		evalNodeName = ann[AnnotationsKeyPrefix+string(sednav1.ILJobEval)]
+		if _, ok := ann[ModelHotUpdateAnnotationsKey]; ok {
```

Comment:
For the synchronization, the corresponding document needs to be modified.

Reply:
have updated doc of example